src/main/java/org/commcare/util/GetJsonProperty.java

```diff
@@ -0,0 +1,24 @@
+package org.commcare.util;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+/** 
+ * @author rcostello
+ * @return A String value for the property name passed in if that property exists else a blank String
+ */
+
+public class GetJsonProperty {
+
+    public static String getJsonProperty(String stringifiedJsonOblect, String propertyName) {
+        JSONObject parsedObject = new JSONObject(stringifiedJsonOblect);
+        String value = "";
+        try {
+            value = parsedObject.getString(propertyName);
```

Comment:
Is this where an exception would get thrown if the given data isn't valid json or isn't an object?

Reply:
Yeah, it throws a JSONException if the key doesn't exist on the object. Going by the spec it seemed likely that an attempt to get a non-existent property would be made at some point.

Reply:
Right, I think I'll need to handle that in the same try block, since it will also throw an exception. "JSONException - If there is a syntax error in the source string or a duplicated key."